client/web/web.go

```diff
@@ -411,6 +374,16 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		data.IP = st.TailscaleIPs[0].String()
 	}
 
+	if asJSON {
+		if err := json.NewEncoder(w).Encode(data); err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
```

Comment:
this is getting logged in output:

```
superfluous response.WriteHeader call from tailscale.com/client/web.(*Server).serveGetNodeData (web.go:383)
```

I think at this point, a 200 response header has already been written